definitions/features/puppet_master.rb

```diff
@@ -0,0 +1,36 @@
+class Features::PuppetMaster < ForemanMaintain::Feature
```

Comment:
I found out, there is also `PuppetServer` feature. Shouldn't both be merged together?
`PuppetServer` is currently just the place for the `services` method that would need to be changed to
```ruby
  def services
    # We only check puppetserver and not puppet-server, as puppet-server
    # is a part of httpd and relies on httpd service to restart, therefore
    # not requiring a separate service to restart
    find_package('puppetserver') ? { 'puppetserver' => 30 } ? {}
  end